proxy/src/main/java/com/wavefront/agent/AbstractAgent.java

```diff
@@ -571,10 +571,16 @@
       "on for jaeger thrift formatted data over TChannel protocol. Defaults to none.")
   protected String traceJaegerListenerPorts;
 
+  @Parameter(names = {"--applicationJaeger"}, description = "Application name for Jaeger. Defaults to Jaeger.")
```

Comment:
Let's rename these parameters to start with "trace" to maintain some consistency. Maybe make it a bit more descriptive too? i.e. traceJaegerApplicationName / traceZipkinApplicationName